http-server/src/main/java/io/micronaut/http/server/binding/BasicAuthBinderRegistrar.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2017-2021 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.http.server.binding;
+
+import io.micronaut.context.event.BeanCreatedEvent;
+import io.micronaut.context.event.BeanCreatedEventListener;
+import io.micronaut.core.annotation.Internal;
+import io.micronaut.http.bind.RequestBinderRegistry;
+import jakarta.inject.Singleton;
+
+/**
+ * A binder registrar that registers Reactive related binders.
+ *
+ * @author Sergio del Amo
+ * @since 3.0.0
+ */
+@Singleton
+@Internal
+class BasicAuthBinderRegistrar implements BeanCreatedEventListener<RequestBinderRegistry> {
```

Comment:
This doesn't need its own registrar. There is already one for http-server. It can just be registered there

Reply:
Which `BinderRegistrar` already exists in `http-server`? I don't find it.

Reply:
it is in `http`